plugin/pom.xml

```diff
@@ -50,6 +50,18 @@
       <groupId>org.everit.json</groupId>
       <artifactId>org.everit.json.schema</artifactId>
       <version>1.5.1</version>
+      <scope>test</scope>
```

Comment:
Adding this to the test scope sounds logical to me, I am not quite sure what `commons-beanutils` does.